sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/QueryStageExec.scala

```diff
@@ -140,6 +140,8 @@ case class ShuffleQueryStageExec(
       case _ =>
     }
   }
+
+  def canChangeNumPartition: Boolean = plan.canChangeNumPartition
```

Comment:
do we need this? It only saves typing 4 characters....

Reply:
Removed it.